swift/extractor/translators/ExprTranslator.cpp

```diff
@@ -624,7 +624,7 @@ codeql::AppliedPropertyWrapperExpr ExprTranslator::translateAppliedPropertyWrapp
 codeql::RegexLiteralExpr ExprTranslator::translateRegexLiteralExpr(
     const swift::RegexLiteralExpr& expr) {
   auto entry = createExprEntry(expr);
-  auto pattern = expr.getRegexText();
+  auto pattern = expr.getParsedRegexText();  // TODO: there is now this and getRegexToEmit
```

Comment:
`getRegexText` was removed in https://github.com/swiftlang/swift/pull/76979.

Looks like the ASTDumper and ASTPrinter have the same replacement as we do here. In `SILGenApply` it gets replaced with `getRegexToEmit`. To me following what the dumper and printer do intuitively makes most sense given what we do with the result (put it in a database). Is that a correct observation?

Reply:
Yeah, I agree. I think we should put what the user has written, rather than what the compiler decides to use internally.